lib/Controller/StackController.php

```diff
@@ -107,4 +107,13 @@ public function delete($stackId) {
 	public function deleted($boardId) {
 		return $this->stackService->fetchDeleted($boardId);
 	}
+
+	/**
+	 * @NoAdminRequired
+	 * @param $boardId
+	 * @return \OCP\Deck\DB\Board
+	 */
+	public function clone($stackId, $boardId) {
+		return $this->stackService->clone($stackId, $boardId, $this->userId);
```

Comment:
Please don't forget our friends from the andorid app and provide a rest api as well ;)